tutorials/Tutorial of QA for FAST.ipynb

```diff
@@ -0,0 +1,287 @@
+{
+ "cells": [
```

Comment:
Here's an outline of what I'm looking for with your tutorial:

qa_fast tutorial
[description of FAST and what your qa does]

Import packages
["qa_fast depends on numpy, nibabel, scipy and uses functions from ndmg's qa_utils"...]
[cell containing all the imports you need for your qa]

qa_fast_png
["qa_fast_png works by (describe what function does in a couple sentences)"]
[ cell containing qa_fast_png code (not commented out) ]

Inputs
[description of each input it needs]
[code you already have under "Set the input data path"]

Call function to generate quality analysis picture
[what you already have here is good]

Potential Issues
[couple sentences on what can go wrong (wm, gm, csf have the wrong color, nothing shows up, common mistakes, etc.)]